apis/core/v1beta1/openstackcontrolplane_webhook.go

```diff
@@ -155,6 +155,11 @@ func (r *OpenStackControlPlane) checkDepsEnabled(name string) string {
 		if !((r.Spec.Mariadb.Enabled || r.Spec.Galera.Enabled) && r.Spec.Memcached.Enabled && r.Spec.Keystone.Enabled) {
 			reqs = "MariaDB or Galera, Memcached, Keystone"
 		}
+	case "Octavia":
+		if !((r.Spec.Mariadb.Enabled || r.Spec.Galera.Enabled) && r.Spec.Memcached.Enabled && r.Spec.Rabbitmq.Enabled &&
+			r.Spec.Keystone.Enabled && r.Spec.Neutron.Enabled && r.Spec.Glance.Enabled && r.Spec.Nova.Enabled) {
+			reqs = "MariaDB or Galera, Memcached, RabbitMQ, Keystone, Glance, Neutron, Nova"
+		}
```

Comment:
In my testing of Octavia itself, I found that I did not need Nova and Neutron to at least get a `OctaviaAPI` pod to run.  I understand though that those services might be required for actually using the deploy Octavia service once the API is available.

Also, I found that I _did_ need to have OVN deployed for my `OctaviaAPI` pod to even deploy.  Otherwise the Octavia operator got stuck on this:

```
ERROR	Reconciler error	{"controller": "octaviaapi", "controllerGroup": "octavia.openstack.org", "controllerKind": "OctaviaAPI", "OctaviaAPI": {"name":"octavia-api","namespace":"openstack"}, "namespace": "openstack", "name": "octavia-api", "reconcileID": "58f3020a-d6df-42a5-a25b-60c48f3ec36e", "error": "OVNDBCluster.apps \"No OVNDBCluster object found in namespace openstack\" not found"}
```

Reply:
Good catch. I've added a check for Ovn. The other services will be required once the amphora-controllers work merges.